resources/views/author.blade.php

```diff
@@ -50,12 +16,13 @@
             @if(isset($books))
             @foreach($books as $book)
             <div class="book-card">
-                <a href="/books/{{$book->book_ID}}" class="book-link">
+                <a href="{{url('books/' . $book->book_ID)}}" class="book-link">
```

Comment:
foloste named routes, te rog. numele route-ului se schimba mai rar decat route-ul in sine (de obicei)